File: kata/6kyu/multiples-of-3-or-5/multiples-of-3-or-5.js
// https://www.codewars.com/kata/514b92a657cdc65150000006

const solution = (number) => {
  let sum = 0
  for (let i = 1; i < number; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
      sum += i
    }
  }

  return sum
}

solution(10) // -> 23 (3 + 5 + 6 + 9)

File: kata/6kyu/multiples-of-3-or-5/multiples-of-3-or-5.py
# https://www.codewars.com/kata/514b92a657cdc65150000006

def solution(number):
    sum = 0;
    for num in range(number):
        if num % 3 == 0 or num % 5 == 0:
            sum += num
    return sum

solution(10) # ->  23